### ISSUE TITLE
Downloading Images

### ISSUE THREAD

**Issue by Jamwali** [microscopium/microscopium#180]
Is there anyway we could download the images as well when we download the selected data ?

**Comment by jni** [comment]
Yes, we should be able to add a "download images" button next to the download table. You can see how the button is added to the layout here:

https://github.com/microscopium/microscopium/blob/db7da4391a694052a6a63273721de05721683ac3/microscopium/serve.py#L363

implemented here:

https://github.com/microscopium/microscopium/blob/db7da4391a694052a6a63273721de05721683ac3/microscopium/serve.py#L261-L275

using some javascript here:

https://github.com/microscopium/microscopium/blob/master/microscopium/js/download_data.js

I have to say I'm not super sure about how to implement it, as you'd want a bit of Python first — to bundle them in a zip file, for example, and serve the zip file.

I did a quick search of the [Bokeh discourse](https://discourse.bokeh.org) and found the ["BokehGarden" tool](https://discourse.bokeh.org/t/new-bokeh-extensions/6963/2) that implements a download button:

https://github.com/emerald-geomodelling/BokehGarden/blob/master/examples/grib_info_download.py

So potentially we can use that? 🤷 

@Jamwali I don't have a lot of time to investigate right now, so I wonder if you think you'd be up for making a pull request to add this feature?
